topics/metabolomics/tutorials/lcms-data-import/tutorial.md

```diff
@@ -23,10 +23,10 @@ contributors:
 
 **TODO** Explain why metabolomics, what do you want to do
 
-To illustrate this approach, we will use data from [Thévenot et al, 2015](https://pubs.acs.org/doi/10.1021/acs.jproteome.5b00354). The objectives of this paper was to analyze the inﬂuence of age, body mass index, and gender on the urine metabolome. To do that, the authors collected samples from 183 employees from CEA and did
+To illustrate this approach, we will use data from {% cite Thvenot2015 %}. The objectives of this paper was to analyze the inﬂuence of age, body mass index, and gender on the urine metabolome. To do that, the authors collected samples from 183 employees from CEA and did
```

Comment:
I think I have understood the idea.
Do you want me to create a `topics/metabolomics/tutorials/lcms-data-import/tutorial.md`?

Reply:
oh, it looks like I forgot to include that one. Thanks, found a couple more I forgot to add :)